src/nvim/ascii.h

```diff
@@ -87,4 +90,45 @@
 # define PATHSEPSTR     "/"
 #endif
 
+static inline bool ascii_iswhite(int c) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
```

Comment:
I think this change in naming might confuse people porting upstream Vim patches. Getting used to `vim_*` was replaced by `ascii_*` is much easier.